docs/source/deployment/knative.rst

```diff
@@ -178,7 +197,7 @@ For this guide, our kubernetes cluster run on minikube, we will get the appropri
     31871
 
 
-With the ip address and port, we can make a curl request to the prediction result from Knative
+With the ip address and port, makes a curl request to the prediction result from Knative
```

Comment:
nit: With the IP address and port, Use `curl` to make an HTTP request to the deployment in Knative: